particles/Native/Kotlin/src/build.gradle

```diff
@@ -0,0 +1,207 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinNativeCompile
+import org.jetbrains.kotlin.gradle.utils.NativeCompilerDownloader
+import org.jetbrains.kotlin.konan.KonanVersion
+
+import java.nio.file.Paths
+
+
+buildscript {
+    ext.serialization_version = "0.12.0"
+
+    repositories {
+        mavenCentral()
+        jcenter()
+
+        maven { url "https://dl.bintray.com/jetbrains/kotlin-native-dependencies" }
+        maven { url "https://kotlin.bintray.com/kotlin-eap" }
+        maven { url "https://kotlin.bintray.com/kotlin-dev" }
+        maven { url "https://polymerlabs.bintray.com/arcs" }
+        afterEvaluate {
+            maven {
+                url "https://kotlin.bintray.com/kotlinx"
+                credentials {
+                    username = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER') ?: ""
+                    password = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY') ?: ""
+                }
+            }
+        }
+    }
+    dependencies {
+        classpath "com.moowork.gradle:gradle-node-plugin:$gradle_node_version"
+        classpath "org.jetbrains.kotlin:kotlin-frontend-plugin:0.0.45"
+        classpath "org.jetbrains.kotlin:kotlin-serialization:0.12.0-SNAPSHOT"
+    }
+}
+
+// see ../settings.gradle so this plugins could be resolved
+plugins {
+    id 'kotlin-multiplatform'
+    id 'kotlinx-serialization'
+}
+
+repositories {
+    mavenCentral()
+    maven { url "https://kotlin.bintray.com/kotlinx" }
+    maven { url "https://polymerlabs.bintray.com/arcs" }
+}
+
+group 'arcs'
+version '0.0.1'
+
+apply plugin: 'maven-publish'
+apply plugin: 'kotlin-dce-js'
+apply plugin: 'org.jetbrains.kotlin.frontend'
+
+kotlinFrontend {
+    downloadNodeJsVersion = "latest"
```

Comment:
perhaps worth tying to a specific version?

Reply:
It's not really used except to run utility code  I think.